CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.25.2 (2025-01-07)
```

Comment:
remove release line. all new features are added to "unreleased" section. I add release and date when I publish - maintaining semantic version based on the set of changes.